test/pom.xml

```diff
@@ -229,13 +254,32 @@ THE SOFTWARE.
         <artifactId>maven-surefire-plugin</artifactId>
         <!-- version specified in grandparent pom -->
         <configuration>
+          <argLine>${jacocoSurefireArgs} -Xmx1g @{jenkins.addOpens} @{jenkins.insaneHook}</argLine>
```

Comment:
Rather than setting these ourselves in a custom profile, we now rely on `maven-hpi-plugin` to set them for us, the same way plugins will in the next release of `plugin-pom`.